README.md

```diff
@@ -1362,6 +1362,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Nur Akmal Mohd Nordi](https://github.com/nurakmal920207)
   - [Nwoke Emmanuella](https://github.com/Ellah120)
   - [Nworie Chikwado Emmanuel](https://github.com/chikwado17)
+  - [Naveenchand](https://github.com/Naveenchand06)
```

Comment:
Please add your name in line 1322, after Nathan Byrne.
Let us know if you need any help with that. 🙂